src/main/java/training/commands/MoveElbowByPosition.java

```diff
@@ -0,0 +1,41 @@
+package training.commands;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import org.littletonrobotics.junction.Logger;
+import training.subsystems.Arm.Elbow.ElbowConstants;
+import training.Robot;
+import training.subsystems.Arm.Elbow.Elbow;
+
+public class MoveElbowByPosition extends Command {
+    private final Elbow elbow;
+
+    private final Rotation2d position;
+    public MoveElbowByPosition(Robot robot, Rotation2d position) {
+        this.elbow = robot.getElbow();
+        addRequirements(elbow);
+        this.position = position;
+    }
+
+    @Override
+    public void initialize() {
+
+    }
+
+    @Override
+    public void execute() {
+        elbow.moveToAngle(position);
+        Logger.recordOutput("difference", Math.abs(elbow.getPosition().getRotations() - position.getRotations()));
```

Comment:
consider either removing this or making a better name for it, like `Elbow/PID Difference`